src/alert.py

```diff
@@ -307,13 +329,21 @@ def alert_nagios(self, checkobjs, debug):
                 msgs[metric] = self.custom_message(metric, results[metric])
                 if msgs[metric] is None:
                     msgs[metric] = self.mc.message(metric, _formats[metric][0], _formats[metric][1])
+  
+        if not msgs:
+            msgs['no_ntp_proc'] = ('CRITICAL: No NTP process could be found.'
+                '  Please check that an NTP server is installed and running.')
+            
```

Comment:
Like you say, I think we need to find an elegant method to do this without duplicating the error messages which are already produced under other circumstances.